packages/text-bitmap/src/BitmapText.ts

```diff
@@ -161,7 +184,7 @@ export class BitmapText extends Container
      *
      * @private
      */
-    updateText()
+    public updateText(): void
```

Comment:
This and `updateTransform` should be private or internal (blank) if used publicly?

Reply:
Oh yea, I think `updateTransform` needs to be blank and this can be private.

Reply:
I think I spotted a couple of others which needed changing too, have updated them.